src/test/unleash.mock.ts

```diff
@@ -15,6 +15,7 @@ class FakeUnleash extends Unleash {
     // fix constructor
     constructor(unleashConfig: UnleashConfig) {
         super(unleashConfig);
+        super.destroy(); // prevent parent constructor initialization
```

Comment:
this line is important as Unleash parent fetches toggles and starts metrics reporting that we don't want